### PR TITLE
Check-in on pharmacy

### LINKED ISSUE #471: Independent Pharmacy: Missing Check-in button
Actual Results
--
On the pharmacy module, the check-in button is not available.

Expected results
--
On the Pharmacy module, once you search and select a preferred patient, the check-in button is displayed on the patient dashboard.

Steps to reproduce
--
Login to eSaude POC->Pharmacy->search for a patient(Milagrasa Nilsa)->click on the patient

Tested on dev server version 2.0.0-alpha2

Screenshot/Attachment (Optional)
--
ip check in error

### PR DESCRIPTION
**Pull request by ynurmahomed**
Fixes #471.

### CODE REVIEW

**Comment by valvijo** [comment]
@ynurmahomed I'm using superuser with all the roles. And aviamento is still open by without checkin

File: app/pharmacy/views/dashboard.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 <visit-header patient="vm.patient"></visit-header>
 
+<check-in patient="vm.patient" on-check-in="vm.reload();"></check-in>
```
Comment (ynurmahomed) [inline app/pharmacy/views/dashboard.html]: `on-check-in` binding was changed to `on-check-in-change` in https://github.com/esaude/esaude-emr-poc/pull/540.

**Comment by ynurmahomed** [comment]
@valvijo updated

**Comment by valvijo** [comment]
It works now! But pharmacy checkin is not enabling Add social form.
image

**Comment by ynurmahomed** [comment]
Might be related to this https://github.com/esaude/esaude-emr-poc/issues/440 was it fixed?

**Comment by valvijo** [comment]
I found the social form Add button issue. Will create a PR for that

**Pull request merged**